pvmismatch/pvmismatch_lib/pvcell.py

```diff
@@ -142,6 +142,18 @@ def Isat1(self):
         )
         return self.Isat1_T0 * _Tstar * _expTstar  # [A] Isat1(Tcell)
 
+    @property
+    def Isat2(self):
+        """
+        Diode two saturation current at Tcell in amps.
+        """
+        _Tstar = self.Tcell ** 3. / self.pvconst.T0 ** 3.  # scaled temperature
```

Comment:
Cool! Nice work!!! Looks like a paper for you. But I think the exponent here should be `5/2` power, not third peer, right? You should consult with a cell researcher like Seung Rim. Also my bad code, you'll save a flop by doing division first then power. Ditto for `Isat1`.

Reply:
Thanks for taking a look Mark! It seems there are various versions of that formula floating around. I took it from 
https://works.bepress.com/dr_adel72/78/ 

Anyhow, I think the Insel slides are more recent and probably a better way to go. But I will consult Seung regarding what'd be more appropriate. Adding references also seems like a good idea. 

Now, I just need to make the travis tests pass!